cf_xarray/tests/test_accessor.py

```diff
@@ -1306,15 +1306,22 @@ def test_Z_vs_vertical_ROMS() -> None:
 
 
 def test_decode_vertical_coords() -> None:
+    # needs standard names on `eta` and `depth` to derive computed standard name
+    romsds.h.attrs["standard_name"] = "sea_floor_depth_below_geopotential_datum"
+    romsds.zeta.attrs["standard_name"] = "sea_surface_height_above_geopotential_datum"
+
+    romsds.cf.decode_vertical_coords(outnames={"s_rho": "z_rho"})
+
+    assert romsds.z_rho.shape == (2, 30)
+    assert romsds.z_rho.attrs["standard_name"] == "height_above_geopotential_datum"
+
+    romsds.drop_vars("z_rho")
+
     with pytest.raises(
         AssertionError, match="if prefix is None, outnames must be provided"
     ):
         romsds.cf.decode_vertical_coords()
 
-    # needs standard names on `eta` and `depth` to derive computed standard name
```

Comment:
why did this need to be deleted.

Reply:
Wasn't deleted just moved to the top of the function.